gravitee-apim-e2e/api-test/src/imports/api-import-update-with-plans.spec.ts

```diff
@@ -35,8 +35,8 @@ describe('Update API by importing it', () => {
     describe('Update API with plans without ID', () => {
       const expectedApiId = '2ce4fa7c-8c75-31a2-83a9-73ccc6773b13';
 
-      const fakePlan1 = PlansFaker.aPlan({ name: 'test plan 1', description: 'this is a test plan', order: 1 });
-      const fakePlan2 = PlansFaker.aPlan({ name: 'test plan 2', description: 'this is a test plan', order: 2 });
+      const fakePlan1 = PlansFaker.plan({ name: 'test plan 1', description: 'this is a test plan', order: 1 });
```

Comment:
All changes in this file are related to https://github.com/gravitee-io/gravitee-api-management/pull/2143/files#r913925890